plane/source/docs/automatic-landing.rst

```diff
@@ -4,52 +4,79 @@
+To land the plane you need to add a :ref:`NAV_LAND <mav_cmd_nav_land>` command to the end of your mission indicating the latitude, longitude and altitude of your desired touchdown point. When the preceeding waypoint is reached, it will steer to the touchdown point.
+In most cases, the altitude should be set to 0. During landing, the autopilot will shut down the throttle and hold the current heading when the plane reaches the flare point, controlled by the parameters described below.
 
-To land the plane you need to add a :ref:`NAV_LAND <mav_cmd_nav_land>` command to the end of your mission indicating the latitude, longitude and altitude of your desired touchdown point. 
-In most cases, the altitude should be set to 0. 
-During landing, the autopilot will shut down the throttle and hold the current heading when the plane reaches the flare point, controlled by the parameters described below.
+For many vehicles, most of the parameters associated with autolanding can be left at their defaults. The basic parameters are covered below in the Basic Autolanding Section for vehicles not using an airspeed sensor and/or rangefinder. For more advanced configuration, see the sections listed below:
 
-.. _automatic-landing_key_parameters:
+.. toctree::
+   :maxdepth: 1
 
-Key Parameters
---------------
+    Aborting an Autolanding <aborting-autolanding>
+    Precision Autolanding <precision-autolanding>
+    RangeFinder Autolanding <rangefinder-autolanding>
+    Using Reverse Thrust in Autolanding <reverse-thrust-autolanding>
+    Using DO_LAND_START <do-land-start>
+
+Basic Autolanding
+=================
+
+The phases of an autolanding are:
+
+- Passing final approach waypoint
+- Navigating to the flare point on a "glide-slope" (ie controlled descent)
+- Flare and touchdown
+
+.. image:: ../../../images/autolanding.jpg
+   :target: ../../_images/autolanding.jpg
+
+Setting Up the Approach Waypoint
+--------------------------------
+
+The autolanding begins after reaching the last navigation waypoint. This waypoint must be far enough from the touchdown point and at an altitude that a reasonable glide slope can be obtained. Without an airspeed sensor, the speed will be at least that of normal waypoint navigation and usually higher, due to the descent. So having a glide slope (angle between last waypoint and touchdown altitude) that is not only obtainable within the TEC_SINK_MAX rate of the vehicle, but well below that rate to avoid overspeeding which makes adjusting the flare more difficult, is required.
+
+In addition, it must be far enough away that any turns made by the mission to that waypoint, will have already be completed and the vehicle back on track in a line to the landing point. Usually, two waypoints are used. The first to allow any mission item (including aborted go-arounds for an autoland reattempt) to get aligned to another waypoint setup closer to the touchdown point but still providing a nice glide slope.
```

Comment:
"it" is a bit vague here, best be clear that "the final approach waypoint must be far enough away" ...

You are referring to the "final approach waypoint" and the "pre-approach waypoint" in several places, including in the diagram, but using different terms for them each time, it would be much clearer if the same wording was used every time.

Reply:
not sure that they are inconsistent

Reply:
At line 41 you call it "last navigation waypoint", whereas here you call it "another waypoint" and in the diagram and on line 48 it's "final approach waypoint"x2.
In the diagram you refer to the "pre-approach" waypoint which on line 43 you refer to as "the first" of two waypoints. Then on line 48 it's called "pre-approach waypoint" again. 
Suggest "The pre-approach waypoint first, to allow any mission item to get aligned to the final approach waypoint ...